schema/scripts/schemas_consolidation/schemas_consolidation.py

```diff
@@ -115,6 +115,7 @@ def parse_api_search(url: str, api_url: str) -> pd.DataFrame:
                     else:
                         filename = res["url"].split("/")[-1]
                     ext = filename.split(".")[-1]
+                    detected_mime = res.get("extras", {}).get("check:headers:content-type", "").split(";").trim()
```

Comment:
Not sure what you are expecting from this: `res.get("extras", {}).get("check:headers:content-type", "").split(";")` returns a list, what is `trim`?

Reply:
My bad https://github.com/etalab/datagouvfr_data_pipelines/pull/29/commits/e58feaa6e9a67553d1c97a47050188c99f7f5f27